catalyst_voices/lib/widgets/common/simple_tree_view.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_shared/catalyst_voices_shared.dart';
+import 'package:flutter/material.dart';
+
+// Note:
+// This widget does not support multi-level nesting yet because
+// it was too complex to implement while we'll have
+// option from google (https://pub.dev/packages/two_dimensional_scrollables)
+// which will be hopefully fixed by time we'll need more nesting.
+class SimpleTreeView extends StatelessWidget {
+  final Widget root;
+  final List<Widget> children;
+  final bool isExpanded;
+
+  const SimpleTreeView({
+    super.key,
+    this.isExpanded = false,
+    required this.root,
+    this.children = const [],
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        root,
+        if (isExpanded) ...children,
+      ],
+    );
+  }
+}
+
+class SimpleTreeViewRootRow extends StatelessWidget {
+  final List<Widget> leading;
+  final VoidCallback? onTap;
+  final Widget child;
+
+  const SimpleTreeViewRootRow({
+    super.key,
+    this.leading = const [],
+    this.onTap,
+    required this.child,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final textTheme = theme.textTheme;
+    final color = theme.colorScheme.primary;
+
+    final textStyle = (textTheme.titleSmall ?? const TextStyle()).copyWith(
+      color: color,
+    );
+
+    final iconTheme = IconThemeData(size: 24, color: color);
+
+    return DefaultTextStyle(
+      style: textStyle,
+      maxLines: 1,
+      overflow: TextOverflow.ellipsis,
+      child: IconTheme(
+        data: iconTheme,
+        child: ConstrainedBox(
+          constraints: const BoxConstraints.tightFor(height: 32),
+          child: Material(
+            type: MaterialType.transparency,
+            child: InkWell(
+              onTap: onTap,
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16),
+                child: Row(
+                  children: <Widget>[
```

Comment:
<Widget> type not needed, in general specifying the type of the list where list of widgets is expected was required very long time ago, noways we skip it... Are you having some issues with linter?

Reply:
actually you're right. In this case it's not needed, even with `separatedBy`